puppetboard/static/css/puppetboard.css

```diff
@@ -44,15 +44,15 @@ h1.ui.header.no-margin-bottom {
   color: #AA4643;
 }
 
-.ui.label.failed {
+.ui.label.failed, .ui.label.events.failure {
```

Comment:
Going to grey by default.

IMO, it's fine this way. Cover relevant colors and default to additionnals.